### DISCUSSION TITLE
How is this profitable?

### DISCUSSION THREAD

**Discussion by QwertyIsCoding** [is-a-dev/register#1677] [Q&A] [answered]
This project is awesome and very useful but I just have a doubt about how this can be profitable?

**Comment by phenax** [comment, marked as answer]
There's no need to make a profit with this as the upkeep is pretty inexpensive (around $50 a year) and takes about a couple of hours every week merging prs, at most. We also had a few maintainers who took some time off their day to review prs.

This project started with me wanting a domain for my personal website but I thought I might as well buy it for everyone. The goal always was to have a completely free (no strings attached) and transparent (domains in this repo are a 1-1 representation of the server) service for registering subdomains and maybe even a framework for other similar services to use to do the same.

**Reply by QwertyIsCoding** [reply]
Oh, nice! Glad to see this.

**Reply by satr14washere** [reply]
50 a year is still a lot 

also how much donors do you have?
one of ur maintainers said 0

**Reply by wdhdev** [reply]
I don't think we have any donators, but as Phenax said, it's quite cheap to maintain.

$50 USD in most first-world countries isn't really a lot, although it might be different in yours.
